src/CLU/Microsoft.ScenarioTests.CLU/out.txt

```diff
@@ -0,0 +1,33 @@
+SETLOCAL EnableExtensions
+
+ECHO Test return code success case
+call testclu success show 2> out.error
+REM We expect that the commands above succeed (return 0)
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO "Expected error level 0 for success, got " %ERRORLEVEL%
```

Comment:
No, I actually, don’t ☺

From: Yugang Wang [mailto:notifications@github.com]
Sent: Thursday, December 17, 2015 4:20 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Johan Stenberg <Johan.Stenberg@microsoft.com>
Subject: Re: [azure-powershell] Update tests + escape paramters passed to clurun by (bash) wrapper script (#1519)


In src/CLU/Microsoft.ScenarioTests.CLU/out.txt<https://github.com/Azure/azure-powershell/pull/1519#discussion_r47980704>:

> @@ -0,0 +1,33 @@

> +SETLOCAL EnableExtensions

> +

> +ECHO Test return code success case

> +call testclu success show 2> out.error

> +REM We expect that the commands above succeed (return 0)

> +IF %ERRORLEVEL% NEQ 0 (

> +     ECHO "Expected error level 0 for success, got " %ERRORLEVEL%

you want to dump out the out.err?

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1519/files#r47980704>.